app/src/main/res/layout/fragment_info.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+xmlns:tools="http://schemas.android.com/tools"
+android:layout_width="match_parent"
+android:layout_height="match_parent"
+tools:context="com.mobileproto.hireddit.hireddit.visuals.InfoFragment"
+android:background="@color/colorBackground"
+android:paddingTop="15dp"
+android:paddingLeft="15dp"
+android:paddingRight="15dp"
+android:paddingBottom="50dp">
+
```

Comment:
@greneroom @YuzhongHuang 
Do you guys want this? I'll add it, if most of want it, but I guess it feels a bit like it's just taking up space. And when I looked at other apps it looks like the upper left back is to signify going 'out a level' versus back, and I don't think leaving the info page is going 'out a level'.